### DISCUSSION TITLE
می‌تونیم یه کارگروه تشکیل بدیم تا این ایده رو عملی کنیم؟

### DISCUSSION THREAD

**Discussion by unlikely-alice** [WeAreMahsaAmini/Core#498] [Ideas]
# استراتژی جمهوری اسلامی برای مقابله با اعتراضات و راه‌کار شکست آن

بر اساس آخرین [نظرسنجی گمان](https://gamaan.org/fa/2023/02/04/protests_survey_persian) که در بهمن ۱۴۰۱ منتشر شد، حدود ۱۹ درصد مردم ایران هنوز به جمهوری اسلامی رأی منفی نداده‌اند. سرپایی جمهوری اسلامی وابسته به این ۱۹ درصد ‌می‌باشد و همه‌ی تصمیمات انجام شده ‌نشان می‌دهد استراتژی جمهوری اسلامی در مرحله اول حفظ این گروه از مردم است.

اگر فرض کنیم، اکثر موافقان جمهوری اسلامی قشر مذهبی‌ای هستند که منبع اصلی خبری آن‌ها رسانه‌های تحت کنترل جمهوری اسلامی است، می‌توان نتیجه گرفت این افراد معتقند که اکثر مردم خواستار تغییر حکومت نیستند و وقایع اخیر را اقلیتی اغتشاش‌گر تشکیل داده‌است. این اعتقاد تصادفی نیست و جمهوری اسلامی برای ایجاد و حفظ این تفکر تلاش بسیار کرده است. تفکر «اقلیت بودن مخالفین» قابل شکست است که می‌تواند منجر به ریزش طرفداران جمهوری اسلامی شود.

## بررسی تفکر طرفداران: «جمعیت خواستار تغییر رژیم جزو اقلیت مطلق است»

با توجه به تعصب این افراد نسبت به نظام جمهوری اسلامی، اعتقادات دینی، منبع خبری، و تجربیات شخصی آنان، این اعتقاد دور از ذهن نیست. برای درک بهتر به چند نکته باید توجه کرد: 

- اعتراضات بسیار پراکنده بوده و این افراد جمعیت بزرگی یک‌جا ندیده‌اند.
- قشر بزرگی از مردم مخالفت خود را با جمهوری اسلامی به زبان نمی‌آروند.
- شعار‌های شبانه بسیار پراکنده است و تفکر اقلیت بودن مخالفین را در ذهن این افراد تقویت می‌کند. 
- مشاهدات آن‌ها با گزارشات رسانه ملی مطابقت دارد. به اعتقاد آن‌ها جمعیت معرضین روی هم صد هزار نفر هم نمی‌شود.
- به اعتقاد آن‌ها «نیروی سرکوب» فقط برای مقابله با اغتشاش گران هست و کسانی که کشته یا اعدام شدند، مستحق آن بودند و همه‌ی کشور های دیگه هم مانند جمهوری اسلامی با اغتشاش گران رفتار می‌کنند.

جمهوری اسلامی برای حفظ این تفکر تلاش‌های بسیاری می‌کند:

- نمایش تصاویر اغتشاش در رسانه های ملی. حتی اگر این تصاویر ساختگی هم باشند در چشم طرفداران جمهوری اسلامی واقعی هستند.
- نمایش اعتراضات در کشور های غربی و مقابله نیروی امنیتی با معترضین (به خصوص در فرانسه)
- اعتراف‌های اجباری در صداوسیما
- ارسال نیروی سرکوب به مکان‌هایی که جمعیت اعتراضی در حال شکل گیری است
- ایجاد وحشت در بین مردم تا به این جمعیت های اعتراضی نپیوندند (زندان، تیر خوردن، اعدام، سوریه شدن، و ...)
- ارسال نفوذی به شبکه‌های اجتماعی برای اتلاف وقت مخالفین تا نتیجه‌ای در بحث ها شکل نگیرد.
- و ...

تمام این تلاش ها از شکل گیری جمعیت بزرگ اعتراضی جلوگیری می‌کند. جمهوری اسلامی از چنین جمعیتی می‌ترسد چرا که یک راهپیمایی بزرگ می‌تواند منجر به شکست در تفکر ذهنی بسیاری از طرفداران‌اش شود. از دست دادن این ۱۹ درصد برای جمهوری اسلامی بسیار ترسناک است، از این رو با استفاده از نیروی سرکوب، تمام تلاش خود را برای جلوگیری از تشکیل چنین جمعیت هایی می‌کند.

## نقطه شکست

در تاریخ شیعه آمده است که امام علی (جانشین پیامبر اسلام از دید شیعه)، با خلافت خود مخالفت کرد تا زمانی که اکثر مردم او را بخواهند و با اون بیعت کنند. از دید موافقان حکومت و با استناد به تاریخ شیعه و تصمیم امام علی که یکی از چهارده معصومین است، حکومت اسلامی بدون خواست اکثریت مردم فایده ندارد.

اگر بتوانیم به این ۱۹ درصد نشان دهیم که اکثر مردم مخالف جمهوری اسلامی هستند، می‌توان ریزش شدید در بین آن‌ها ایجاد کرد تا جایی که حتی شاید سبب پیروزی انقلاب بدون خشونت شود.

طرفداران حکومت، نتیجه نظرسنجی هایی مانند گمان را نمی‌پزیرند، جمهوری اسلامی اجازه برگزاری رفراندوم را نمی‌دهد، و نیروی سرکوب نیز اجازه شکل گیری راهپیمایی های بزرگ را نمی‌دهد.

باید راهی پیدا کنیم که همه‌ی استراتژی های جمهوری اسلامی را دور بزند و در عین حال مثل روز روشن به تمام طرفداران جمهوری اسلامی نشان دهد که اکثریت مردم این رژیم را نمی‌خواهند.

## راهکار

این راهکار دو مرحله داره. مرحله اول یارگیری و مرحله دوم اثبات حداکثریت به صورت متحد.

دلیل وجود یارگیری اینه که مرحله دوم شاید یه مقدار کوچیکی خطر داشته باشه ولی با استفاده از یارگیری و به کار گیری اصل «یا همه با هم، یا هیچ کس» می‌تونیم خطر مرحله دوم رو به صفر برسونیم.

در مرحله‌ی اول، همه مردم یه عبارت خاص مثل «اتحادیه مردم ایران» را در گوگل هر شب ساعت ۹ جست‌و‌جو می‌کنند و روی لینک یک سایت مشخصی کلیک می‌کنند. با استفاده از گوگل ترندز و پنل گوگل می‌شه فهمید تقریبا چه درصدی از مردم دارن این کار رو انجام می‌دن. تو وبسایت هم نقشه رو توضیح می‌دیم. پس یعنی این درصد از مردم از نقشه خبر دارن و وقتی که به اندازه کافی یار جمع شد، آماده هستند به مرحله دوم برن.

در مرحله‌ی دوم، فقط کافیه ساعت ۹ شب همه با هم از خونه‌ یا هرجای دیگه‌ای که هستند شعار بدن. اگر طرفداران حکومت ببینند که در همه‌ی محلات بالای ۵۰ درصد مردم در حال شعار دادن هستند، دیگه نمی‌تونن استراتژی «اقلیت و اغتشاش» رو قبول کنند و این باعث می‌شه در اون ۱۹ درصد ریزش شدید ایجاد بشه.

یه جست‌وجو ی ساده در گوگل ایده‌ی خوبی هست برای مرحله اول چون:

- می‌تونیم بزرگی اتحادیه رو با استفاده از گوگل ترندز به دست بیاریم.
- فیلتر گوگل هزینه‌ی خیلی بالایی برای جمهوری اسلامی داره. شاید بتونن یکی دو هفته فیلتر کنن ولی دوباره بعدش باید باز کنن و ما هم به کارمون ادامه می‌دیم
- همه‌ی ایرانی ها استفاده از گوگل رو بلد هستند
- لازم نیست تو سایت خاصی ثبت نام کنند، بدون اکانت هم می‌شه تو گوگل جست‌وجو کرد
- کمتر از یک دقیقه وقت ‌می‌گیره و در هر مکانی قابل انجام هست.
- جمهوری اسلامی ‌نمی‌تونه بفهمه چه کسانی این کار رو انجام می‌دن تا اونا رو در شروع کار سرکوب کنه (وقتی که هنوز تعداد زیاد نشده)
- کسایی که می‌خوان تو اعتراضات شرکت کنند ولی ترس دارن هم می‌تونن اینکار رو انجام بدن چون هیچ خطری نداره
- یه زمان مشخص داره که باعث می‌شه نظم رو تمرین کنیم (برای پیروزی باید با هماهنگی و منظم عمل کنیم)
- یه فعالیت روزانه هست و یک بار نیست که بعضی ها یک بار انجام بدن و بعد دو هفته دیگه یادشون بره. باید هر روز انجام بدن تا تعداد به حد بالایی برسه که وارد مرحله دوم بشیم.

**Comment by arandomgstring** [comment]
20% is not a number to laugh at. Take for example, commodity exchange markets. A holder of 20% of shares is literally a god in that particular commodity as long as 80% of shares is divided among many people. To take the power from that holder, either that 20% has to become so much smaller, in some cases less than 1% or something, or someone has to buy more than 20% of shares from the other 80% to gain the upper hand in that commodity. In other words, either an internal conflict inside the government is needed, or majority of people has to unify on what they want after revolution. Both of these scenarios seem to be far fetched at the moment, especially the former one. While your idea might momentarily unify people on protests, it doesn't address the main problem: that is, the purpose of people for such protests. For sure, people want to get rid of this government, but for what alternative? That's the core of issue, as you are already well aware.

I think your idea can be developed to some extent. For example, this website can adapt a natural stance, and become the host and voice of a few well-known characters who want a revolution. These characters should introduce themselves and their ideas thoroughly and furthermore advertise this website on their verified social accounts and ask their followers to advertise even more. Anyone can follow them on this website. After certain number of followers, they can ask their followers to protest at a certain time, or follow some objectives such as donating for something, or perhaps doing something harsher. 

There-afterwards, they can show the result of their objectives with actual documentations. For instance, they can show approximately how many people went into protest with a few blurred videos, or if it was an objective against certain people, what happened, etc. With this method, not only people with different beliefs can do different things against government simultaneously but also they can see which leader does things in reality objectively, rather than having a big mouth with nice words. I think gradually majority of people will be inclined toward a better character & opinion, which is what they need when it comes to revolution. In the meanwhile, the objective of their protests becomes more meaningful. 

P.S: 
Twitter and other social medias are a mess. A website just for something like this is needed. Especially somewhere that only well-known character can voice their opinion. A condition for being well-known can be set, such as number of followers on social medias, having special political power, etc.  
They are not afraid of censoring Google. As they have already zarebin which uses Google API under-hood, and censors its results.

**Reply by unlikely-alice** [reply]
تقریبا همه‌ی حرف هایی که زدی رو قبول دارم.

البته الان ۲۰ درصد موافق جمهوری اسلامی نیستن، فقط ۲۰ درصد رای منفی ندادن. طبق گمان ۱۵ درصد موافق هستند که این درصد هر سال داره کمتر میشه. یعنی شکاف‌هایی که میگی داره داخل موافقین ایجاد میشه. من خودم این رو به چشم دیدم، آدم‌هایی که به هیج عنوان فکر نمی‌کردم به خامنه‌ای شک کنند، الان یه مقدار شک کردن، هنوز طرفدار هستند ولی نه مثل قبل.

ما در کل باید دو تا کار انجام بدیم:
یک. اگر یه طوری نشون بدیم که حداکثریت جمهوری اسلامی رو نمی‌خوان، این شکاف ها بزرگ تر هم میشه. فقط باید یه طوری نشون بدیم که بهونه نتونن بیارن، مثلا بیش از ۵۰ درصد مردم باید تو شعار های شبانه شرکت کنند.
دو. اتحاد بین خودمون رو بیشتر کنیم با حل کردن مشکلات موجود. مثلا در آینده چی می خوایم. چه چیزی جایگزین جمهوری اسلامی باید بشه و از کجا باید شروع بشه.

در مورد وب سایتی هم که گفتی، دارم روش کار می‌کنم. ولی تنهایی نمی‌شه. مشکل اصلی کد زدن نیست. مشکل اصلی تبلیغات و دعوت از متخصصین و این چیزاست. که یه کارگروه لازم داره.

**Reply by arandomgstring** [reply]
Isn't it possible to ask whoever that wants to be hosted on this website in future, to sponsor and advertise it first? I think a prototype of this website would be convincing enough for many so to speak "leaders" to start their help. They can help with donation or advertisements, and the final version of that website will be built from these helps. Of course other interested parties can join even after final version coming out. The only real problem is perhaps keeping this website neutral and prove this neutrality to visitors. It would be best if two opposing parties join.

**Reply by unlikely-alice** [reply]
اگر یه دونه وبسایت مرکزی باشه هیچ جوره نمیشه ثابت کرد که اون وبسایت از قدرتش سوء استفاده نمیکنه. راه حلی که به ذهن من رسید اینه:
- اولا یه دونه وبسایت نباشه. چند تا وبسایت باشه هر کدوم مخصوص یه کاری
- دوما وب سایت ها یک صاحب ندارن. مثلا دامنه برای یکی هست، سرور برای یکی دیگه.
- سوما، همه‌ی وبسایت ها باید روزانه فایل پشتیبان منتشر کنن که اگر تخطی کردن راحت بتونیم از روی فایل پشتیبان یه وبسایت دیگه بیاریم بالا و اون یکی رو دیگه استفاده نکنیم.
می تونیم میزبان ها رو هم حتی یک ماه یکبار عوض کنیم.

البته یه نکته ای. به نظرم میشه متحد شد. لازم نیست هر کدوم از «لیدر» ها جدا عمل کنه. بیشتر اختلافی که بین انقلابیون هست روی اسم هست. هم سلطنت طلب‌ها، هم جمهوری‌خواه‌ها و هم بقیه یه چیزی مثل کشور‌های غربی می‌خوان. هیچ کسی دنبال دیکتاتور بعدی نیست.

**Reply by arandomgstring** [reply]
Advertising a single domain is a pain in and of itself. Making people use different domains is a nightmare. I would rather see a single website which has become a brand than several unknown websites. Besides the existence of several domains under different owners doesn't prove anything, because all of these owners might belong to a specific party anyway. Therefore, I think to keep things as simple as possible, developing a single website would be faster and easier. The problem of neutrality can be tackled as long as two opposing parties join this website. As long as they kept satisfied, it means that the neutral stand of website remains.

However, the idea of backing up the data is excellent. The source code of website can be published on GitHub itself, so that no matter what happens to developers, or owners, it can be brought back by another trusted party. And if there is a security issue or something, it can be found as fast as possible; though it would also means government too has access to these unpatched holes. Regardless of implementation, be it in the GitHub or not, backup is a must. 

You see, if the unification of these parties was as easy as you say, the revolution would have already happened. While there are some bullet points that all parties can agree on such as democracy, separation of religion and politics, etc, we can see that in practice these parties don't merge, maybe because of greed for power, or any other reason that I don't understand. So I thought that developing a platform for this purpose (i.e unification of powers, or if not possible, destroying all parties except one with the most noticeable results) might be a solution to this conundrum.

**Reply by unlikely-alice** [reply]
در مورد پاراگراف اول.
شما می‌گید یه وب‌سایتی باشه مثل تویتر که همه از اون استفاده کنند؟ حتی مردم عادی؟ اون خیلی سخت می‌شه. البته یه سری گروه‌های مختلف دارن روی ایده‌های مشابه کار می‌کنند، مثل [این](https://github.com/tcfev/forDem). ولی یه همچین وبسایتی هم از لحاظ توسعه خیلی کار سختی هست و هم از لحاظ عملیاتی (سرور های قدرتمند لازم داره، صد ها نفر باید تمام وقت روش کار کنند، و ...، یه چیزی مثل تویتر کار راحتی نیست).

در مورد پاراگراف آخر، می‌تونی [اینجا](https://github.com/WeAreMahsaAmini/Core/discussions/499) بیشتر توضیح بدی؟ فقط من یه نکته بگم، موافقت در مورد دموکراسی برای انقلاب کافی هست. راستش نمی‌شه انتظار داشت همه با هم سر همه‌ی مسائل متحد بشن، فقط چند تا مسئله بنیادین مثل چیز‌هایی که خودت گفتی قابل پذیرش در سطح گسترده هست. همه‌ی کشورها همینطوری هستند. نابودی همه‌ی پارتی ها به غیر از یکی، میشه یه دیکتاتور دیگه.

**Reply by arandomgstring** [reply]
No! Of course not. Building a social network like twitter is a nightmare. What I am proposing is that only a handful of well-known people (leaders of new revolution) can publish content on this website. Even if that's hard, still an administrator can get content from these leaders and publish it under their name instead. Normal people can only see these contents and like them so that leaders and people themselves can get a general idea about attitude of majority of people toward certain things. Normal people cannot do anything else, other than seeing and liking, otherwise this website will become a mess, just like twitter itself. I am simply offering content for the very website you mentioned first. Take for example this scenario: a few leaders suggest different times/locations for a protest. People can like these suggestions, and in the end the time and location with the majority of likes would be chosen. Of course, a leader might advocate the idea of parallel protesting, i.e having a lot of small protests at different times, instead of a single large one. This idea too can be put forward to see if people are interested in this type of thing or not. Just a strong anti-bot system against hacking likes is needed. 

I will try. By destroying other parties, I mean we have quite a few democratic systems offered by different parties. It is not possible to satisfy everyone at the same time, in the end a political system is either democratic monarchy or republic. Semi presidential, or presidential, socialist or capitalist, etc. Only one of these system can govern people, not all of them at the same time. Therefore, we have to know, at some point, what majority wants and discard other parties for an ultimate unification. It will not result in an authoritarian regime, in the end, a democratic system will be chosen. The leaders themselves should explain their ideology clearly in simple words for people. Just spitting out "democracy" doesn't make someone/a system democratic. Before Islamic revolution, people were yelling out democracy too, but see what happened? The problem in Islamic republic is not just its political figures, rather this system itself, its constitution, along with its important rules is not (and never can be) democratic in the first place. When you give so much power to a single person (leader of republic), of course your system will become authoritarian! That's why I am saying that before revolution, the purpose of revolution should become 100% clear. Maybe a website dedicated to this helps.
